src/app/archive/lib/processor.ml

```diff
@@ -4745,80 +4745,91 @@ let add_genesis_accounts ~logger ~(runtime_config_opt : Runtime_config.t option)
         With_hash.{ data = block; hash = the_hash }
       in
       let add_accounts () =
-        Mina_caqti.Pool.use
```

Comment:
Could we add some docs for this whole function `add_genesis_accounts`? Here's the current doc which doesn't seem to be informative:

```
(* [add_genesis_accounts] is called when starting the archive process *)
```